docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -2,30 +2,31 @@
 
-  You can set up the component ports to [send and receive requests](#port) between components of component-based application with the `Tizen.Applications.ComponentBased.ComponentPort` class.
+-   Managing component port
 
-- Using component task
+    You can set up the component ports to [send and receive requests](#port) between components of component-based applications with the `Tizen.Applications.ComponentBased.ComponentPort` class.
 
-  You can set up the component tasks to [run tasks](#task) and wait for events from other components with the `Tizen.Applications.ComponentBased.ComponentTask` class.
+-   Using component task
+
+    You can set up the component tasks to [run tasks](#task) and wait for events from other components with the `Tizen.Applications.ComponentBased.ComponentTask` class.
 
 ## Prerequisites
 
-To enable your component to use the component port functionality:
+To enable your component to use the component port functionality, follow these steps:
 
 1.  You need two components to communicate with each other through the component port.
-2.  To use the methods and properties of the [Tizen.Applications.ComponentBased.ComponentPort](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html) and [Tizen.Applications.ComponentBased.ComponentTask](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html) classes, include the [Tizen.Applications.ComponentBased](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.html) namespace in your component:
+2.  To use the methods and properties of the [Tizen.Applications.ComponentBased.ComponentPort](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html){:target="_blank"} and [Tizen.Applications.ComponentBased.ComponentTask](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html){:target="_blank"} classes, include the [Tizen.Applications.ComponentBased](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.html){:target="_blank"} namespace in your component:
 
     ```csharp
     using Tizen.Applications.ComponentBased;
     ```
 
-<a name="local"></a>
-## Managing component port
+<a name="port"></a>
+## Manage component port
 
-To send a request from one component `ClientService.Tizen` to another `ServerService.Tizen` component, use the [Tizen.Applications.ComponentBased.ComponentPort](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html) class as follows:
+To send a request from one component `ClientService.Tizen` to another `ServerService.Tizen` component, use the [Tizen.Applications.ComponentBased.ComponentPort](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html){:target="_blank"} class as follows:
```

Comment:
Just a question.

https://**samsung.github.io**/TizenFX/**API9**/api/Tizen.Applications.ComponentBased.ComponentPort.html


It seems this API link is going to API9, Tizen 6.5.
Also, for .NET, the hyperlinks for API are in /application/dotnet/api/TizenFX/... form.

**Reference :** https://github.com/Samsung/tizen-docs/pull/1483/files#diff-2ce89e0eb4b5b4611274e3a7db61390c9b45eec7c05076b427851c1ef5ad9f92


Would you please check the API links also to make them go to the latest, referring other links?

Reply:
@mijin85cho,
Thank you for pointing it out.
I will change the links accordingly.